app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -51,3 +58,11 @@ export const haveTranslation = operand => translatedOperands.includes(operand)
 export const isEnglishLocale = () => currentLocale() === 'en';
 
 const currentLocale = () => I18n.currentLocale();
+
+export const toggleColumn = (projectBoard, column, callback) => {
+  if (isOpen(projectBoard, column)) {
+    if (canCloseColumn(projectBoard)) return callback.onToggle();
+  } else {
+    return callback.onToggle();
+  }
```

Comment:
Maybe this can be simplified like that:

```js
  if (isOpen(projectBoard, column) && canCloseColumn(projectBoard)) {
    return callback.onToggle();
  }

  return callback.onToggle();
```